simulator-core/src/test/java/com/evbox/everon/ocpp/it/provisioning/ColdBootChargingStationIt.java

```diff
@@ -0,0 +1,64 @@
+package com.evbox.everon.ocpp.it.provisioning;
+
+import com.evbox.everon.ocpp.mock.StationSimulatorSetUp;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.BootNotification;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.StatusNotification;
+import com.evbox.everon.ocpp.mock.ocpp.exchange.TransactionEvent;
+import com.evbox.everon.ocpp.simulator.StationSimulatorRunner;
+import com.evbox.everon.ocpp.simulator.configuration.SimulatorConfiguration;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.OCPP_SERVER_URL;
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.STATION_ID;
+import static com.evbox.everon.ocpp.mock.expect.ExpectedCount.times;
+import static com.evbox.everon.ocpp.mock.factory.SimulatorConfigCreator.createSimulatorConfiguration;
+import static com.evbox.everon.ocpp.mock.factory.SimulatorConfigCreator.createStationConfiguration;
+import static com.evbox.everon.ocpp.v20.message.station.StatusNotificationRequest.ConnectorStatus.AVAILABLE;
+import static org.awaitility.Awaitility.await;
+
+public class ColdBootChargingStationIt extends StationSimulatorSetUp {
+
+    private static String ADDITIONAL_STATION_ID = "EVB-P18090564";
```

Comment:
final? and I think it make sense to move this constant to a shared constant-class

Reply:
done